cs.py

```diff
@@ -186,26 +186,31 @@ def read_config():
     conf = ConfigParser()
     conf.read(paths)
     try:
-        return conf['cloudstack']
+        return conf[ini_group]
     except AttributeError:  # python 2
-        return dict(conf.items('cloudstack'))
+        return dict(conf.items(ini_group))
 
 
 def main():
-    config = read_config()
-
     usage = "Usage: {0} <command> [option1=value1 " \
-            "[option2=value2] ...] [--async] [--post]".format(sys.argv[0])
+            "[option2=value2] ...] [--async] [--post] " \
+            "[--region=<region>]".format(sys.argv[0])
 
     if len(sys.argv) == 1:
         raise SystemExit(usage)
 
     command = sys.argv[1]
     kwargs = defaultdict(set)
-    flags = set()
+    flags = dict()
```

Comment:
Can you use an additional variable for options instead of mixing them with flags? This would leave the flags logic mostly unchanged.